entity/src/main/java/com/wilma/entity/positions/ExpressionOfInterest.java

```diff
@@ -33,37 +33,27 @@ public class ExpressionOfInterest {
     @Column(name = "id", nullable = false)
     private Integer id;
 
-    @ManyToOne
-    @JoinColumn(name = "partner_id")
-    private Partner partner;
+    @Column
+    private String category;
 
-    @Column(name = "start_date")
-    private Date startDate;
-
-    @Column(name = "end_date")
-    private Date endDate;
-
-    private Period period;
+    @Column
     private String location;
-    private String description;
 
-    private boolean filled;
-    private boolean approved;
+    @Column
+    private String description;
+    
+    @Column
+    private Date date;
 
-    @Transient
-    private String type;
-    //Category
+    @Column
```

Comment:
Without declaring a column name, the JPA generated column name for this will be `ISFILLED` 
```suggestion
@Column(name = "is_filled")
```
This will create a column name `IS_FILLED`